corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -912,6 +912,21 @@ def get_reverse_indexed_cases(domain, case_ids, case_types=None, is_closed=None)
         _attach_prefetch_models(cases_by_id, indices, 'case_id', 'cached_indices')
         return cases
 
+    @staticmethod
+    def check_transaction_order_for_case(case_id):
+        """ Returns whether the order of transactions needs to be reconciled by client_date
+
+        True if the order is fine, False if the order is bad
+        """
+        if not case_id:
+            return False
+        with get_cursor(CaseTransaction) as cursor:
```

Comment:
Using a proxy function adds extra layers of complexity and more code to maintain. I think it would be simpler using the approach @snopoke outlined here.